src/components/data-management/LaunchAnalysisButton.jsx

```diff
@@ -68,16 +76,22 @@ const LaunchAnalysisButton = () => {
 
   useEffect(() => {
     // The value of backend status is null for new experiments that have never run
-    const pipeline = isTechSeurat ? 'seurat' : 'gem2s';
-    const pipelineBackendStatus = backendStatus[activeExperimentId]?.status?.[pipeline];
+    const setupPipeline = isTechSeurat ? 'seurat' : 'gem2s';
+    const {
+      pipeline: qcBackendStatus, [setupPipeline]: setupBackendStatus,
+    } = backendStatus[activeExperimentId]?.status ?? {};
 
     if (
-      !pipelineBackendStatus
+      !setupBackendStatus
       || !experiments[activeExperimentId]?.sampleIds?.length > 0
     ) return;
 
-    const pipelineStatus = calculateGem2sRerunStatus(pipelineBackendStatus, activeExperiment);
-    setPipelineRerunStatus(pipelineStatus);
+    const status = calculateGem2sRerunStatus(setupBackendStatus, activeExperiment);
+
+    setPipelinesRerunStatus({
+      gem2s: status,
+      qc: calculateQCRerunStatus(qcBackendStatus, setupBackendStatus, activeExperiment),
```

Comment:
calculating the qc rerunStatus is not needed if the gem2s one needs rerun right?

Reply:
In general you only need to know one thing that needs to be reran following the imporancy:

1. if gem2s needs to be reran just rerun gem2s and do not care about qc
2. if gem2s to is fine but qc needs rerun then just rerun qc
3. Both are fine

Seems like generalising the rerun to a single variable for rerunStatus for gem2s and qc would be simpler and enough